packages/katello/rubygem-foreman_kernel_care/rubygem-foreman_kernel_care.spec

```diff
@@ -1,74 +1,50 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-
 %global gem_name foreman_kernel_care
 %global plugin_name kernel_care
 %global foreman_min_version 1.19.0
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.1.1
+Name: rubygem-%{gem_name}
+Version: 1.2.0
 Release: 1%{?foremandist}%{?dist}
 Summary: Plugin for KernelCare
-Group: Applications/Systems
 License: GPLv3
 URL: https://github.com/maccelf/foreman_kernel_care
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 1.5.6
-Requires: %{?scl_prefix}rubygem(katello) >= 3.8.0
```

Comment:
I am not sure if it is correct that all the dependencies are gone from the spec file.

Reply:
it is, they are now autogenerated